### ISSUE TITLE
Section508.gov - Buy Accessible Products and Services

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#315]
7th most popular search for VPAT - https://www.section508.gov/buy/

Under **Market Research Process** change bullets under 2: 

> - Search the internet for existing Accessibility Conformance Reports (ACR) for the ICT being sought. These are often referred to as [Voluntary Product Accessibility Template](https://www.itic.org/policy/accessibility/vpat) (VPAT™). Look for new ACRs produced with the new industry standard OpenACR format. You will then evaluate these ACRs against the 508 standards for your ICT.
> - Review any existing ACRs or product 

Under **Step 4 - Request Accessibility Information From Vendors** change bullet:
> - For COTS/GOTS, Request an ACR) or other evidence of accessibility testing, for each ICT item that is to be developed, updated, configured for the agency, and when product substitutions are offered. The ACR should be based on OpenACR, although for some projects legacy VPAT™ may be acceptable.

And also
> Refer vendors to [https://section508.gov/sell](https://www.section508.gov/sell) for guidance on using ACRs.

Under **Step 5 - Evaluate Proposals - Accessibility Conformance Report Review**
> - Review the Accessibility Conformance Report (ACR) and/or product to determine how well the product conforms to the Revised 508 Standards. Vendors often post this information on their website, typically on their accessibility or product description page.

Todo
- [ ] Determine if OpenACR should be linked elsewhere
- [ ] Review for plain language

**Comment by mgifford** [comment]
See https://github.com/GSA/Section508.gov/pull/229
